frontend-infrastructure/build.gradle

```diff
@@ -1 +1,134 @@
+import com.amazonaws.blox.tasks.PostProcessCloudformation
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+
 description "AWS Cloudformation templates and deployment scripts for the Blox Frontend API"
+
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'io.swagger:swagger-models:+'
+    }
+}
+
+ext {
+    templateFile = file("cloudformation/template.yml")
+    processedTemplateFile = file("${buildDir}/template.json")
+    outputTemplateFile = file("${buildDir}/template.output.json")
+
+    stackOutputsFile = file("${buildDir}/${stackName}-${awsProfile}.outputs.json")
```

Comment:
Again, I don't think we should be including the profile name in the file name. What's the advantage of doing that?